doc(release): Prepare release notes for web 22.10.22, gorgone 22.10.7 (#3284)
centreon/centreon-documentation: i18n/fr/docusaurus-plugin-content-docs/version-22.10/releases/centreon-os.md

## Centreon Web

### 22.10.22

Release date: `May 16, 2024`

#### Enhancements

> - [Enhancement] unattended.sh script now allows new options (Debian12, MariaDB versionalization, MySQL 8)

Reviewer: ```suggestion
- [Enhancement] The unattended.sh script now offers new options (Debian12, MariaDB versioning, MySQL 8).
```

#### Bug fixes

> - [API] Fixed an issue on downtimes for services linked to service group via service template.

Reviewer: ```suggestion
- [API] Fixed an issue affecting downtimes on services linked to a service group via a service template.
```

> - [Configuration] Fixed an issue in Service Categories configuration page (blank page in some cases).

Reviewer: ```suggestion
- [Configuration] Fixed an issue where the Service Categories configuration page could appear blank.
```

#### Security fixes

> - [Security] Fixed SQLi in configuration.
> - [Security] Fixed SQLi in graph generation.
> - [Security] Fixed SQLi in service configuration.
> - [Security] Fixed SLQi in hosts configuration.
> - [Security] Fixed SQLi in connectors configuration.
> - [Security] Fixed SQLi in hostgroups configuration.
> - [Security] Fixed SQLi in media management.

Reviewer: ```suggestion
- [Security] Fixed several SQLi issues.
```
